.github/workflows/continous-integration-os.patch.yml

```diff
@@ -15,16 +15,18 @@ on:
 
 jobs:
   test:
-    name: Test ${{ matrix.rust }} on ${{ matrix.os }}
-    runs-on: ${{ matrix.os }}
```

Comment:
Did you mean to stop running on the matrix of os's?

Reply:
There isn't a matrix defined in this workflow file but just checking

Reply:
Yep, there's no need to use up one of our 5 macOS runners just to run:
```sh
echo "No build required"
```

https://docs.github.com/en/actions/learn-github-actions/usage-limits-billing-and-administration#usage-limits